### PR TITLE
discovery Log enabled in debug mode

### LINKED ISSUE #100: As a user, I can see the HTTP request content in the logs for Google Services
WHY
Currently, we use the Discovery SDK, which hides a lot of complexity. However, for troubleshooting it could be useful to see the HTTP endpoint and the query body when we use NCK in a debug mode. This is always asked when we contact the support.

HOW
Add a log statement (debug level) before executing the request with the SDK. This log should contain as much information as possible about the request: endpoint, auth method, payload ...

### PR DESCRIPTION
**Pull request by louisRDSC**
### Issue

- [ ] My PR resolves #100 

### Description

- [ ] created 2 decorators allowing https details into logs (1 is for the reader functions the other one is for the __init__)
- [ ] Used these decorators in DV360_reader
- [ ] I added the possibility to set the Log level with an env variable

### CODE REVIEW

File: nck/config.py [hunk 1]
```diff
@@ -20,6 +20,8 @@
 import os
 
 LEVEL = logging.INFO
+if "LOGGING_LEVEL" in os.environ:
+    LEVEL = os.environ["LOGGING_LEVEL"]
```
Comment (benoitgoujon) [inline nck/config.py:24]: One way to handle default values for `dict` is to use the `.get()` method. What do you think about this?
```suggestion
LEVEL = os.environ.get("LOGGING_LEVEL", logging.INFO)
```

File: nck/utils/stdout_to_log.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            self.content = []
+
+    def flush(self):
+        pass
+
+
+def http_log(logger_name, level=logging.DEBUG):
+    def decorator(func):
+        def wrapper(*args, **kwargs):
+            httplib2.debuglevel = 4
```
Comment (benoitgoujon) [inline nck/utils/stdout_to_log.py:37]: Is there a constant with an explicit name that hides this magic number 4. Would be great for readability.

**Review by benoitgoujon — COMMENTED** [review]
Thank you for this contribution 👍 

I think you need to install the pre-commit hooks to follow the code quality guidelines. Lines have been shortened for ex, so use our pre-commit hooks to avoid this.

File: nck/config.py (hunk 1 above)
Comment (louisRDSC) [reply nck/config.py:24]: yes you're right it is shorter and clearer.

File: nck/utils/stdout_to_log.py (hunk 2 above)
Comment (louisRDSC) [reply nck/utils/stdout_to_log.py:37]: No there isn't. But after further checking it seems that any value greater than 0 activate debug mode.
```suggestion
            httplib2.debuglevel = 1
```
Do you prefer it like that?

Comment (benoitgoujon) [reply nck/utils/stdout_to_log.py:37]: In the [documentation](https://httplib2.readthedocs.io/en/latest/libhttplib2.html#httplib2.debuglevel), they say that this is the amount of information we want to print. So, with 1, we may miss some info. Let's keep it that way 🙂

**Review by benoitgoujon — APPROVED** [review]

**Pull request merged**